web/html/src/manager/minion/ansible/ansible-path-type.tsx

```diff
@@ -0,0 +1,11 @@
+export type AnsiblePath = {
+  id: number;
+  minionServerId: number;
+  type: string;
+  path: string;
+}
+
+export function createNewAnsiblePath(fieldsToUpdate: Partial<AnsiblePath>) {
+  const todo: AnsiblePath = {id: -1, minionServerId: -1, type: "", path: ""};
```

Comment:
Just in case you missed this, a todo here, but I suspect this is for the future.

Reply:
LOL, a leftover of a copy-paste from [Partial<Type> example](https://www.typescriptlang.org/docs/handbook/utility-types.html#partialtype)